internal/app/fetcher/libraries.go

```diff
@@ -0,0 +1,77 @@
+package fetcher
+
+import (
+	"context"
+	"github.com/pkg/errors"
+	"github.com/xssnick/tonutils-go/tlb"
+	"github.com/xssnick/tonutils-go/tvm/cell"
+)
+
+type LibDescription struct {
+	_          tlb.Magic        `tlb:"$00"`
+	Lib        *cell.Cell       `tlb:"^"`
+	Publishers *cell.Dictionary `tlb:"dict inline 256"`
+}
+
+func (s *Service) GetAccountLibraries(ctx context.Context, raw *tlb.Account) (*cell.Cell, error) {
+	hashes, err := findLibraries(raw.Code)
+
```

Comment:
It would be nice to remove all redundant spaces, before code blocks like the following

```golang
if err != nil {
```


```golang
if ok {
```